src/validations/rules/ssp.sch

```diff
@@ -3866,16 +3866,6 @@
             doc:context="/o:system-security-plan/o:control-implementation/o:implemented-requirement"
             id="DNSSEC-described-diagnostic">The implemented requirement does not contain the strings 'DNSSEC' or 'DNS Security
             Extensions'.</sch:diagnostic>
-        <sch:diagnostic
-            doc:assertion="contingency-planning-for-information-system-backup"
-            doc:context="/o:system-security-plan/o:control-implementation/o:implemented-requirement"
-            id="contingency-planning-for-information-system-backup-diagnostic">The statement <xsl:value-of
-                select="@statement-id" /> does not contain correct interval terms for system backups.</sch:diagnostic>
-        <sch:diagnostic
-            doc:assertion="contingency-planning-for-information-system-backup2"
-            doc:context="/o:system-security-plan/o:control-implementation/o:implemented-requirement"
-            id="contingency-planning-for-information-system-backup2-diagnostic">The statement <xsl:value-of
-                select="@statement-id" /> does not contain correct scope terms for system backups.</sch:diagnostic>
```

Comment:
Why was this not removed when the related assertions were removed?

Reply:
This section of code is not in the ssp.sch.